server/profileFormSubmission.js

```diff
@@ -8,25 +11,52 @@ const pool = new Pool({
     port: process.env.port,
 });
 
+s3.config.update({
+    accessKeyId: "AKIA47HST523YVS7DQNW",  
+    secretAccessKey: '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',
+    region: "us-west-1",
+});
+
+const LICENSE_BUCKET = "cleaner-licenses";
+const TOOLS_BUCKET = "cleaner-tools";
```

Comment:
Maybe instead of having a bucket per front-end form field, we can have a bucket per security level? Ex: confidential vs non-confidential instead of license & tools
That might help keep the buckets flexible to handle future front-end form changes

Reply:
Yeah, that's a great idea, I started to implement it but AWS was being really weird and not letting me make a new bucket for the confidential stuff. I'll try again tomorrow.